pynitrokey/cli/storage.py

```diff
@@ -49,5 +49,120 @@ def enable_update(password):
         local_print("No Nitrokey Storage device found")
 
 
+@click.command()
+@click.option(
+    "-p",
+    "--password",
```

Comment:
We have to decide on a policy for password input, see also <https://github.com/Nitrokey/pynitrokey/issues/194>.  My suggestion would be to use an environment variable as the default or a stdin prompt if the variable is not set.  @szszszsz thoughts?

Reply:
Sounds good to me. So something like the below, but instead moved to a custom handler for better reusability:
- https://click.palletsprojects.com/en/7.x/options/#dynamic-defaults-for-prompts

After #194 I encourage to block passing any passwords through the option switch unless some force flag or environment variable is passed (to allow usage in tests, like allow_dangerous or in the spirit of ALLOW_ROOT).

Reply:
I don’t like the idea of enabling an option with another option or with an environment variable.  I think that’s more likely to confuse users who don’t know about that mechanism.  I’d prefer to directly use an environment variable to set the password (e. g. `NITROPY_STORAGE_USER_PIN` and `NITROPY_STORAGE_ADMIN_PIN`).  What do you think about that?

Reply:
@robin-nitrokey Since these could be used for NK Pro too, shouldn't we call it more generic NITROpy_USER_PIN for example?

Reply:
Agreed with the general naming.

Regarding my point, I just want to safekeep the potentially user-hostile features, which should not be used in production, but in tests only (and thus should be a bit complicated by design). On the other hand I do not have a problem using env variables in tests at all, so in the end I would opt to remove the PIN passing through option/switch completely, and ask for it with prompt if it was not found in the env variables.

Reply:
@szszszsz Yes, that’s what I meant – no option, only stdin or environment variable.
@jans23 I suggested the more verbose variant to reduce the risk of accidentally setting the wrong variable.  Especially when taking into account the other subcommands (fido2, nk3, nethsm), I think it’s important to make the variable names unambiguous.

Reply:
@robin-nitrokey In all our documentations these PINs are called User PIN and Admin PIN. Adding "STORAGE" to it makes it more confusing, at least to me.

Reply:
@jans23 I see your point.  It was supposed to read “User PIN for `nitropy storage`”, not ”Storage User PIN for `nitropy`”.  So would you be okay with using the same variables for different products?

Reply:
Yes. I think using the same variables for different (OpenPGP Card-compatible) products would be the least confusing.

Reply:
I'm not sure I understand what ENV variable we should use in the end. `USER_PIN` and `ADMIN_PIN`?

Reply:
I’d keep the `NITROPY_` prefix to avoid collisions with other tools, so `NITROPY_USER_PIN` in this case.